### PR TITLE
chore: allow to provide other class parameters on the component

### LINKED ISSUE #6407: allow to provide class parameters to Badge component
### Is your enhancement related to a problem? Please describe

For the work on extensions page I need to provide some settings to the Badge component but it's not possible

### Describe the solution you'd like

allow to propagate class

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by benoitf**
### What does this PR do?
usually we need to provide some classes to place the component as expected

### What issues does this PR fix or reference?

Fixes #6407 

### How to test this PR?

- [x] Tests are covering the bug fix or the new feature

### CODE REVIEW

**Review by axel7083 — APPROVED** [review]
LGTM

File: packages/renderer/src/lib/ui/Badge.spec.ts [hunk 1]
```diff
@@ -89,3 +89,13 @@ test('Should display badge with dark color', async () => {
...
 });
+
+test('Should display badge with custom class', async () => {
+  render(Badge, { label: 'customLabel', class: 'text-right' });
+
+  // expect to see label inside the label
+  const label = screen.getByText('customLabel');
+  expect(label).toBeInTheDocument();
+
+  vi.waitFor(() => label.classList.contains('text-right'));
```
Comment (axel7083) [inline packages/renderer/src/lib/ui/Badge.spec.ts]:
```suggestion
  vi.waitFor(() => expect(label).toHaveClass('text-right'));
```

Comment (axel7083) [reply packages/renderer/src/lib/ui/Badge.spec.ts]: Shouldn't you be waiting for the vi.waitFor ? it is not blocking

Comment (benoitf) [reply packages/renderer/src/lib/ui/Badge.spec.ts]: yes it was testing nothing

should be fixed now 👍

**Review by axel7083 — APPROVED** [review]
Re-approving

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**